docs/extensions/api.md

```diff
@@ -26,34 +28,51 @@ other issues on the [bug tracker].
 
 ## Preprocessors {: #preprocessors }
 
-Preprocessors munge the source text before it is passed into the Markdown
-parser. This is an excellent place to clean up bad syntax, extract things the
-parser may otherwise choke on and perhaps even store it for later retrieval.
+Preprocessors munge the source text before it is passed to the Markdown
+parser. This is an excellent place to clean up bad encodings or to extract 
```

Comment:
We defiantly want "syntax" not encodings" here. Markdown only accepts Unicode text, therefore any encodings will have been lost by time time preprocessors get the text. Yes, I realize you might be speaking of something else, but I don't want anyone to misunderstand.

Reply:
Removed encodings; you are correct.  It now says 'bad characters' which is accurate.

Reply:
But it still needs to say "syntax." That was intentional.